extensions/3rdparty/ArrayExtension/ArrayExtension.php

```diff
@@ -411,7 +411,7 @@ function arrayunique( &$parser, $arrayid ) {
 *          http://www.php.net/manual/en/function.shuffle.php
 *          http://us3.php.net/manual/en/function.array-reverse.php
 */
-    function arraysort( &$parser, $arrayid , $sort = 'none') {
+    function arraysort( $parser, $arrayid , $sort = 'none') {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the unused function parameter "$parser". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)